etna/search/tests/test_views.py

```diff
@@ -519,9 +529,10 @@ def test_render_invalid_date_range_message(self):
             "<li>Try removing any filters that you may have applied</li>", content
         )
         self.assertSearchWithinOptionRendered(content)
-        self.assertIn(
-            "<li>There is a problem. Start date cannot be after end date.</li>", content
+        content_error_message_count = content.count(
+            f'<ul class="errorlist nonfield"><li>{CUSTOM_ERROR_MESSAGES.get("invalid_date_range")}</li></ul>'
```

Comment:
Could we keep this simple and just test for the list item with hardcoded text? I know hardcoding the test is slightly more fragile, but the value is hardly likely to change (and if it did, the error would be expected)

Reply:
Done. [here](https://github.com/nationalarchives/ds-wagtail/blob/f17e917ada03a73c8a6062a5abaadf9156cf579d/etna/search/tests/test_views.py#L629-L632). Also updated the ul with id for each set of date as they are non field errors.